src/GUI/ReverseEngineer20/DacpacConsolidate/DacpacConsolidator.cs

```diff
@@ -54,6 +54,14 @@ private List<string> GetAllReferences(string dacpacPath, bool isRootDacpac)
                 .Select(i => Path.GetFullPath(Path.Combine(Path.GetDirectoryName(dacpacPath), i.Value)))
                 .ToList();
 
+            if (!isRootDacpac)
```

Comment:
Do we not already handle this in DcapacMerger line 39?

Reply:
Indeed, I didn't see that one.
Could either remove this, or keep it to "fail fast" and remove missing paths earlier.
Sadly I forgot the exact reason I put that there, or rather why the line in DacpacMerger didn't solve the issue I had.